affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt

```diff
@@ -125,6 +126,7 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
         )
     }
 
+    @Suppress("UnstableApiUsage")
     @VisibleForTesting
     internal fun registerInternalTask(
```

Comment:
I suppose that we must to call this code in `registerCustomTasks` also

Reply:
Added